### PR TITLE
More Survival Tools Series 9

### PR DESCRIPTION
**Pull request by chaosvolt**
1. Added makeshift sunglasses, basically the Inuit snow goggles. owo

### CODE REVIEW

**Comment by DavidKeaton** [comment]
nice!

**Comment by chaosvolt** [comment]
I would also note that a few of the cartridges available historically originated in the blackpowder era. I didn't add the option to them because it would be a bit odd to run blackpowder cartridges through autoloading guns. o3o

File: data/mods/More_Survival_Tools/armor.json [hunk 1]
```diff
@@ -263,5 +263,31 @@
...
+        "symbol" : "[",
+        "description" : "Eyewear carved from from wood or bone, with only a thin slit to see out of. Helps reduce glare, traditionally used by the Inuit to protect against snow blindness.",
+        "price" : 4500,
+        "material" : ["bone", "null"],
+        "volume" : 1,
+        "cutting" : 0,
+        "warmth" : 10,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
```
Comment (Coolthulhu) [inline data/mods/More_Survival_Tools/armor.json]: For something that leaves just a thin slit to see through, this sounds way too low.

File: data/mods/More_Survival_Tools/recipes.json [hunk 2]
```diff
@@ -1588,5 +1588,358 @@
...
+[
+[ "chem_saltpetre", 75 ]
+],
+[
+[ "chem_sulphur", 10 ]
+] ,
+[
+[ "charcoal", 15 ]
+]
+]
```
Comment (Coolthulhu) [inline data/mods/More_Survival_Tools/recipes.json]: Looks like your formatting got nuked here for some reason

**Comment by Coolthulhu** [comment]
Since gunpowder is going to become uncraftable, you could push gunpowder as a separate PR into mainline.

**Comment by Rivet-the-Zombie** [comment]
Continually adding all these things into a mod is only going to make it into a longer process when we pull them into the mainline. Why not just PR your stuff for the mainline in the first place?

**Comment by Zireael07** [comment]
Good point.

**Comment by chaosvolt** [comment]
Meep. Maybe should PR the blackpowder idea then. =w=

Also, I should note I copy-pasted the blackpowder recipe from the recipe file, so if the formatting is FUBAR it wasn't my mistake. :V

**Comment by chaosvolt** [comment]
Swiggity swooty: https://github.com/CleverRaven/Cataclysm-DDA/pull/12656

**Comment by chaosvolt** [comment]
Meep. And thank you for the merge. ^^

**Pull request merged**